server/sonar-server/src/main/java/org/sonar/server/issue/index/IssueIndex.java

```diff
@@ -177,6 +172,160 @@ public IssueIndex(EsClient client, System2 system, UserSession userSession, Auth
     this.sorting.addDefault(IssueIndexDefinition.FIELD_ISSUE_KEY);
   }
 
+  /**
```

Comment:
Not sure it was a good idea to reformat the class in this commit as it's now super hard to see what has changed...